operations/image_management/Upload_and_Register_an_Image_Recipe.md

```diff
@@ -131,3 +131,145 @@ Download and expand recipe archives from S3 and IMS. Modify and upload a recipe
     type = "s3"
     ```
 
+## Templating IMS recipes when building an IMS image
```

Comment:
Change to:

Templating IMS Recipes when Building an IMS Image

Reply:
Done